pkg/3scale/amp/component/apicast.go

```diff
@@ -142,7 +125,7 @@ func (apicast *Apicast) StagingDeploymentConfig() *appsv1.DeploymentConfig {
 					ServiceAccountName: "amp",
 					Volumes:            apicast.stagingVolumes(),
 					Containers: []v1.Container{
-						v1.Container{
+						{
 							Ports:           apicast.stagingContainerPorts(),
 							Env:             apicast.buildApicastStagingEnv(),
 							Image:           "amp-apicast:latest",
```

Comment:
Should we set the image?

Reply:
I kept it because that's how the DeploymentConfig had it configured but I don't think we have to set it. Either way it will be changed due to the ImageStreamTag annotation - see [docs](https://docs.openshift.com/container-platform/4.14/openshift_images/triggering-updates-on-imagestream-changes.html).